### ISSUE TITLE
rg.exe CPU overflow 

### ISSUE THREAD

**Issue by delfws** [microsoft/vscode#78597]
![image](https://user-images.githubusercontent.com/24715927/62589849-20eab480-b8fd-11e9-980a-a40cf4e370d1.png)

when i open the file, i try to scroll the page.but the rg.exe has been create too much.just look like this.

i try to search it in the baidu. i find to set "search.followSymlinks: false" and restart the vscode.but it has unuseful

**Comment by roblourens** [comment]
Are you doing a search? rg.exe is triggered by something searching. Does it happen if you run code with `--disable-extensions`?

**Comment by delfws** [comment]
> Are you doing a search? rg.exe is triggered by something searching. Does it happen if you run code with `--disable-extensions`?

For the first time, I tried to uninstall some of the plugins, because I suspect that it is because of the plugin conflict, and finally found that the vetur plugin will cause this problem, but I need this plugin. So I tried the second way, uninstalling other plugins, because my vscode is the default update plugin by default, it may be because other plugin updates lead to compatibility issues between versions, so after I uninstall some useless plugins, he is normal. However, I have not been able to find the real cause of the problem. It may be that a plugin will cause frequent scanning of files after automatic update.
